appcelerator/titanium_mobile
# feat: update template for angular 9

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-27856

**Optional Description:**

Updated template for Titanium Angular with support for the new Webpack build pipeline and Angular 9.

**Test case:**

1. Create a new Titanium Angular app with `ti create --template angular-default` or `appc new --ng`
2. Build and run the app
3. Verify that the example works. You should see a very simple app with a welcome screen shown on first launch. Tapping on the button opens a second screen, which is the main view of the app. The main screen shows a simple data binding example with a button and a label. Clicking the button will update the label text, and reset after 5 taps.
